components/ui/AvatarIcon.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="avatar-icon">
+    <svg
```

Comment:
This is horrendous. It can be achieved with DIVs and positioning.

Reply:
Do you mean the SVG in general? I am using it to mask the image, but you are right, it isn't very intuitive and requires a vector editor to change it. I could just not use an SVG and instead make a border around the red bubble that matches the header color.

Reply:
No more SVG with 5fb6d9d, but choosing the correct border color for the notification bubble required some extra CSS and a prop. Though, overall, I do think it is simpler.